underactuated/exercises/grader.py

```diff
@@ -52,9 +52,9 @@ def grade_output(test_case_list, notebook_locals_list, results_json):
     def locals_from_notebook(notebook_ipynb):
         """Read, run, return locals of notebook"""
         banned_commands = [
-            'start_recording',
-            'stop_recording',
-            'get_recording_as_animation',
+            # 'start_recording',
+            # 'stop_recording',
+            # 'get_recording_as_animation',
```

Comment:
In general, the advice is to not leave dead code in comments, but do delete it.  Any reason to keep these?

Reply:
I agree. I just kept them for the case that we would want to reactivate them in the future. I removed them now.